bats/gql/api-keys.gql

```diff
@@ -3,6 +3,7 @@ query apiKeys {
     defaultAccount {
       id
     }
+    scopes
```

Comment:
`scopes` on `me` evokes the idea that this user / account only ever has permission for those scope. But in actual fact it changes depending on the authentication method (which api key for instance).
So I think this needs to be semantically distinguished as its a property of the authentication method.
Ideas are:
`authenticatedScopes`
`currentScopes`
`authentication { scopes }`

Or perhaps something outside of the `me` query

wdyt @vindard

Reply:
I like the direction of `authentication { scopes }` because then you can eventually add things like `type` (api key, bearer token etc.), `apiKey` (to share some details about the requesting key) etc.

Reply:
I also agree that putting the scope into `me` means we would be merging the concept of `token` and `user`

right now, `me` is really a synonym or `user` so we have:

```
me (user)
  account
    wallet
````

really the token is adding a new layer. this would be away to think about it:
```
token
  user
    account
      wallet
```

the reason token now is a different entities is 2 folds:
- authorization can be different
- there might be budget associated with a token

the budget is probably one of the most important part going forward, and a highly requested features. 

but I don't think we want `budget` to fall under the concept of a `user`, but rather a `token` or `authentication`

Reply:
I think we need another top level query for this information (not nested under `me`).

It is feasible that an api token that is making the call can make / receive payments but _not_ have authorisation to read other user data.

how about:
```
Query {
  authorization {
    scopes
  }
}
```

Or something like that.